CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+* The `Plugin.__call__` method now supports a `cleanup` argument
```

Comment:
I suggest adding a brief description also in online documentation. This will be important option when trying to debug calculations.